packages/auth/src/AuthSignIn/AuthSignIn.tsx

```diff
@@ -13,15 +15,29 @@ export type AuthSignInProps = {
 };
 
 const AuthSignIn = ({ onSignIn, onSignUp, defaultValues }: AuthSignInProps) => {
+  const { formatMessage } = useIntl();
+
   const schema = yup.object().shape({
     email: yup
       .string()
-      .required('Custom required message')
-      .email('Custom email message'),
+      .required(
+        formatMessage({ id: 'auth.signIn.field.email.messages.required' })
+      )
+      .email(
+        formatMessage({
+          id: 'auth.signIn.field.email.messages.validEmail',
```

Comment:
@RayzaOliveira from [this link](https://formatjs.io/docs/getting-started/message-declaration), we have:

> While you can declare your messages using only ids, we highly recommend declaring defaultMessages inline along with their usages because of the following reasons:

I've already worked using ids and it doesn't scale. Do you have a strong reason to use ids?

Reply:
I thought about doing them at first and then refactoring them to a more scalable model. I don't have any specific reason for using ids, they were just my first steps.